tests/metrics/test_isolated_label.py

```diff
@@ -57,3 +57,16 @@ def test_isolated_labels_perfect(adata_pca):
     )
     LOGGER.info(f"score: {score}")
     assert_near_exact(score, 1, diff=1e-12)
+
+
+def test_isolated_labels_asw_perfect(adata_pca):
+    adata_pca.obsm["X_emb"] = _random_embedding(partition=adata_pca.obs["celltype"])
+    score = scib.me.isolated_labels_asw(
+        adata_pca,
+        label_key="celltype",
+        batch_key="batch",
+        embed="X_emb",
+        verbose=True,
+    )
+    LOGGER.info(f"score: {score}")
+    assert_near_exact(score, 0.9983715917167036, diff=1e-12)
```

Comment:
must have missed adding a test for the ASW isolabel score 🙈 Thanks for adding that.

Might be better to compare with reference score 1 and adjust the tolerance accordingly (since this is supposed to represent a perfect score)